src/ert/gui/ertwidgets/checklist.py

```diff
@@ -119,7 +119,7 @@ def filterList(self, _filter):
             item = self._list.item(index)
             text = str(item.text()).lower()
 
-            if _filter == "" or _filter in text:
+            if not _filter or _filter in text:
```

Comment:
Here this change will make it trigger on `_filter` being `None` or `0` in Python as there is no type. But since we have called `_lower()` anything else but a string will make it crash beforehand so the change is ok. It would be nice to have `_filter` typed though.

Reply:
I think that if we also want to assert that the type is string, we should do isinstance(str,...) instead. I assume the old idea was that it would only check if the filter was set.